src/app/core/components/profile-edit/profile-edit.component.ts

```diff
@@ -1,18 +1,36 @@
 import { Component, OnInit, Inject, AfterViewInit } from '@angular/core';
```

Comment:
```suggestion
/* @format */
import { Component, OnInit, Inject, AfterViewInit } from '@angular/core';
```
Since Prettier was run on this file, can we add the `@format` pragma so this file is checked by Prettier?